pycbc/gate.py

```diff
@@ -77,7 +77,6 @@ def apply_gates_to_td(strain_dict, gates):
     # copy data to new dictionary
     outdict = dict(strain_dict.items())
     for ifo in gates:
-        logging.info("Gating {} strain".format(ifo))
```

Comment:
Landscape says we've got an unused import. Remove the ``import logging`` line now.

Reply:
Done.